src/cmdstan-guide/pathfinder_intro.Rmd

```diff
@@ -0,0 +1,129 @@
+```
+Path [1] :Initial log joint density = -11.543343
+Path [1] : Iter      log prob        ||dx||      ||grad||     alpha      alpha0      # evals       ELBO    Best ELBO        Notes 
+              5      -6.748e+00      1.070e-03   1.707e-05    1.000e+00  1.000e+00       126 -6.220e+00 -6.220e+00                  
+Path [1] :Best Iter: [5] ELBO (-6.219833) evaluations: (126)
+Path [2] :Initial log joint density = -7.443345
+Path [2] : Iter      log prob        ||dx||      ||grad||     alpha      alpha0      # evals       ELBO    Best ELBO        Notes 
+              5      -6.748e+00      9.936e-05   3.738e-07    1.000e+00  1.000e+00       126 -6.164e+00 -6.164e+00                  
+Path [2] :Best Iter: [5] ELBO (-6.164015) evaluations: (126)
+Path [3] :Initial log joint density = -18.986308
+Path [3] : Iter      log prob        ||dx||      ||grad||     alpha      alpha0      # evals       ELBO    Best ELBO        Notes 
+              5      -6.748e+00      2.996e-04   4.018e-06    1.000e+00  1.000e+00       126 -6.201e+00 -6.201e+00                  
+Path [3] :Best Iter: [5] ELBO (-6.200559) evaluations: (126)
+Path [4] :Initial log joint density = -8.304453
+Path [4] : Iter      log prob        ||dx||      ||grad||     alpha      alpha0      # evals       ELBO    Best ELBO        Notes 
+              5      -6.748e+00      2.814e-04   2.034e-06    1.000e+00  1.000e+00       126 -6.221e+00 -6.221e+00                  
+Path [4] :Best Iter: [3] ELBO (-6.161276) evaluations: (126)
+Total log probability function evaluations:8404
+```
+
+Pathfinder output is written into a set of StanCSV files; one file which contains the importance resampled draws
+from multi-path Pathfinder, and one file which contains the output of each single-path Pathfinder run.
+Given an output file name, CmdStan adds suffixes to the base filename to distinguish between the output files.
+For the default output file name `output.csv`, the resulting set of CSV files are
+```
+output_pathfinder.csv
+output_1.csv
+output_2.csv
+output_3.csv
+output_4.csv
+```
+
+The file `output_pathfinder.csv` contains the importance-resampled draws; the other output files contain the draws from the single-path Pathfinder runs.   For both sets of outputs, the organization of information is the same.
+The initial CSV comment rows contain the complete set of CmdStan configuration options.
+Next is the column header line, followed the set of approximate draws.
+
+The pathfinder algorithm first outputs `lp_approx__`, the log density in the approximating distribution, and `lp__`,  the log density in the target distribution, followed by estimates of the model parameters, transformed parameters, and generated quantities.
+```
+lp_approx__,lp__,theta
+-2.4973, -8.2951, 0.0811852
+-0.87445, -7.06526, 0.160207
+-0.812285, -7.07124, 0.35819
+
+...
+```
+
+The final lines are comment lines which give timing information.
+```
+# 
```

Comment:
I'd remove 117, 121, 126, and 129, even though they're in the output file.  It's just comments.